easybuild/easyblocks/s/suitesparse.py

```diff
@@ -94,8 +94,12 @@ def configure_step(self):
         try:
             for line in fileinput.input(fp, inplace=1, backup='.orig'):
                 for (k, v) in cfgvars.items():
```

Comment:
Please be consistent in your code review, there's another one like it on on 109.